### ISSUE TITLE
is zgen dead?

### ISSUE THREAD

**Issue by AdrienLemaire** [tarjoilija/zgen#123]
Hi @tarjoilija,

Is there a roadmap for zgen and features being developed?  Or has support for this project been dropped?
If you cannot maintain this anymore, should we move over to zplug?

Asking because I see no commits in 7 months and no answers in the issues.

**Comment by wadkar** [comment]
It works ✅
Stable enough ✅ 
That sounds good enough for me.

What features would you like to be developed for zgen? Can you create a PR/issue for the same? I wouldn't mind looking into it and I am sure @tarjoilija will be happy to accept reasonable PRs.

**Comment by mkcode** [comment]
This project looks dead to me.

Not only has there been nothing committed, but no issue or PR comments on many good PRs. @tarjoilija last commit to this project and the whole of github was on Oct 20, 2018

**Comment by AdrienLemaire** [comment]
Yes, I've switched to antibody https://github.com/getantibody/antibody

**Comment by rjcoelho** [comment]
Don't know if it's dead but I have moved to https://github.com/zimfw/zimfw/tree/develop

**Comment by ratijas** [comment]
Pretty much dead by now. It's time to archive the repo.

**Comment by jandamm** [comment]
Hi, I've resurrected zgen in my fork [zgenom](https://github.com/jandamm/zgenom).
Among other things, I've added these features:
- compiling plugins & lazy loading
- support for executables `zgenom bin`
- full support for head branches instead of `master`

Please check out the [full list](https://github.com/jandamm/zgenom#new-features).

It's also worth noting that zgenom is fully backwards compatible so you don't need to change your zshrc. Have a look at the [migration guide](https://github.com/jandamm/zgenom#migration-from-zgen).
I hope you like it 🙂

**Comment by marlonrichert** [comment]
> Yes, I've switched to antibody

Antibody is officially dead now, too. If anyone is interested in finding a new plugin manager that's light-weight, _fast_ and easy to use, I can highly recommend [Znap](https://github.com/marlonrichert/zsh-snap). It can reduce your time to prompt to ~40ms with just one command. 🙂

**Comment by ratijas** [comment]
> > Yes, I've switched to antibody
> 
> Antibody is officially dead now, too. If anyone is interested in finding a new plugin manager that's light-weight, _fast_ and easy to use, I can highly recommend [Znap](https://github.com/marlonrichert/zsh-snap). It can reduce your time to prompt to ~40ms with just one command. slightly_smiling_face

I've set up a countdown until znap also declares end of life. ;-)

In the end it turned out, if you know how to download files and manage your own dotfiles repository, you don't really need a plugin manager for zsh.

**Comment by jandamm** [comment]
@ratijas I totally agree.
But not everybody has to reinvent the wheel or has the time to do so.

With [zgenom](https://github.com/jandamm/zgenom) I don't do anything you couldn't do yourself with git, file operations and a bit of zsh knowledge.
In fact previously I used zgen with a few hand-rolled scripts. (I didn't like submodules)

If you have some tricks up your sleeve I'd be happy to see a contribution or an issue from you 😉
